doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
+<a name="1.3"></a>
+## 1.3 Project Goals and Purpose
+The goal of the task force is to develop a robust infrastructure model and a limited discrete set of architectures built on that model that can be tested and validated for use across the  entire member community. The community, which is made up of a cross section of global operators and supporting vendors alike, was created to support the development, deployment and management of NFV applications faster and more easily.  
 
-This nature of this disaggregation is what gives the para-virtualized interfaces the flexibility that makes them favourable in a virtualized environment.
+All of this had led to a growing awareness of the need to develop more open models and validation mechanisms to bring the most value to telco operators as well as vendors, by agreeing on a standard set of infrastructure profiles to use for the underlying infrastructure to support VNF applications across the industry and telecom community at large. To achieve this goal, the NFVI environment needs to be fully abstracted via APIs and other mechanisms to the VNFs so that both developers of the VNF applications and the operators managing the environments can benefit from the flexibility that the disaggregation of the underlying infrastructure offers.
 
-The downside of para-virtualization interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+The next step after the NFVI Reference Model has been identified and developed is to take the general model, which is purposely designed to be able to be applied to a number of technologies, and apply it to a discrete number of concrete and ultimately deployable Reference Architecture platforms. The intention is to chose the reference architectures carefully so that there will only be a small set of architectures that meets the specific requirements for supporting NFV and Telecom specific applications. Per the principles laid out in the Reference Model documentation, the Reference Architectures need to meet the following criteria as much as is practical:
 
-<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 4:</b> Para-Virtualized interface components (software only).</p>
+  - Initially should be based on widely established technology and systems used in the Telecom Industry.  This will help ensure a faster adoption rate because the operators are already familiar with the technology and might even have systems in production. Another advantage to this approach is a project faster development cycle.
+  - Subsequent architectures should be based on either additional established or promising emerging technologies that are chosen by the community members.  
 
-<a name="4.1.2"></a>
-#### 4.1.2 Direct assignment with IOMMU.
+<a name="1.4"></a>
+## 1.4 Common NFVI Benefits
```

Comment:
Given the discussion in #1168 can I suggest changing `Common NFVI` to `Common Cloud Infrastructure`?  Will need to update at the top of the page too.

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/df3e4fdb211f993433dacb9ed86f6b621c9e8ef9